config/queue.php

```diff
@@ -84,4 +84,5 @@
         'table' => 'failed_jobs',
     ],
 
+    'backoff' => explode(',', env('QUEUE_BACKOFF', '10,60,300,900,1500')), // 10s, 1m, 5m, 15m, 30m
```

Comment:
If someone would set QUEUE_BACKOFF to the empty string, would this use the default? I'm wondering if we should have a null default so that we can turn this behavior on and off by means of configuration.

Reply:
Good thinking, I removed the values